_opensearch/supported-field-types/xy-point.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: xy point
+nav_order: 58
+has_children: false
+parent: Cartesian field types
+grand_parent: Supported field types
+---
+
+# xy point field type
+
+An xy point field type contains a point in a two-dimensional Cartesian coordinate system, specified by x and y coordinates. It is based on the Lucene [XYPoint](https://lucene.apache.org/core/8_5_1/core/org/apache/lucene/geo/XYPoint.html) field type. The xy point field type is similar to the [geopoint]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point/) field type, but does not have the range limitations of geopoint. The coordinates of an xy point are single-precision floating point values. For information about the range and precision of floating-point values, see [Numeric field types]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/numeric/).
+
+## Example
+
+Create a mapping with an xy point field type:
+
+```json
+PUT testindex1
+{
+  "mappings": {
+    "properties": {
+      "point": {
+        "type": "xy_point"
+      }
+    }
+  }
+}
+```
+
+## Formats
+
+xy points can be indexed in the following formats:
+
+- An object with an x and y coordinate
+
+```json
+PUT testindex1/_doc/1
+{
+  "point": { 
+    "x": 0.5,
+    "y": 4.5
+  }
+}
+```
+
+- A string in the "`x`, `y`" format
+
+```json
+PUT testindex1/_doc/2
+{
+  "point": "0.5, 4.5" 
+}
+```
+
+- An array in the [`x`, `y`] format
+
+```json
+PUT testindex1/_doc/3
+{
+  "point": [0.5, 4.5] 
+}
+```
+
+- A [Well-Known Text](https://docs.opengeospatial.org/is/12-063r5/12-063r5.html) POINT in the "POINT(`x` `y`)" format
+
+```json
```

Comment:
Shall we include xy point as GeoJSON as well? This feature will be introduced in 2.4 as well. 
ex:
```
{
  "point" : {
    "type" : "Point",
    "coordinates" : [0.5, 4.5]        
  }
}
```

Reply:
Added the GeoJSON format.